lib/authn.js

```diff
@@ -170,11 +156,22 @@ function _checkServerVersion({serverInfo, config}) {
     url: urls.sanitize(config.url),
     version,
   });
+  // if the server is not at least 3.6.0 throw
+  // as mongo node driver 4 only supports 3.6.0 and higher
+  if(!semver.gte(version, '3.6.0')) {
+    throw new BedrockError(
+      'Unsupported database version.',
+      'NotSupportedError', {
+        url: urls.sanitize(config.url),
+        version,
+        required: '>= 3.6.0'
```

Comment:
Isn't a different version being required by code changes elsewhere in this PR?

Reply:
The driver only goes down to 3.6 now: https://www.mongodb.com/docs/drivers/node/current/compatibility/#mongodb-compatibility

So 3.6 is the hard limit on server version.